src/components/event/details/DetailsSuspectedCommits.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <DetailsBase
+    class="details-suspected-commit"
+    :expandShowed="commits.length > numberOfVisibleCommits"
+    @expandClicked="isMoreCommitsShown = !isMoreCommitsShown"
+  >
+    <template #header> {{ $t("event.suspectedCommits") }} </template>
+    <template #content>
+      <div
+        v-for="(commit, index) in filteredCommits"
+        :key="index"
+        class="event-details__content-block details-suspected-commit__content-block"
+        data-ripple
+      >
+        <div class="details-suspected-commit__left">
+          <div class="details-suspected-commit__left-title">
+            {{ commit.title }}
+          </div>
+          <div>
+            <span class="details-suspected-commit__left-author">
+              {{ commit.author }}
+            </span>
+            <span class="details-suspected-commit__left-relative-time">
+              commited {{ commit.date | prettyRelativeTimeStr }}
+            </span>
+          </div>
+        </div>
+        <div class="details-suspected-commit__right">
+          <div class="details-suspected-commit__right-hash-block">
+            <span class="details-suspected-commit__right-hash">
+              {{ commit.hash.substr(0, 7) }}
+            </span>
+          </div>
+        </div>
+      </div>
+    </template>
+    <template #expandButton>
+      {{
+        isMoreCommitsShown
+          ? "Hide"
+          : `${
+              commits.length - numberOfVisibleCommits
+            } more commits in this release`
+      }}
+    </template>
+  </DetailsBase>
+</template>
+
+<script>
+import DetailsBase from './DetailsBase';
+
+export default {
+  name: 'DetailsSuspectedCommits',
+  components: {
+    DetailsBase,
+  },
+  props: {
+    /**
+     * Event commits to show
+     */
+    commits: {
+      type: Array,
+      required: true,
+    },
+    
+  },
+  data() {
+    return {
+      /**
+       * Is block expanded.
+       */
+      isMoreCommitsShown: false,
+      /**
+       * Number of Visible Commits
+       */
+      numberOfVisibleCommits : 1,
+    
+    };
+  },
+  computed: {
+    /**
+     * Displayed commits items
+     */
+    filteredCommits() {
+      return this.commits.length <= this.numberOfVisibleCommits ||
+        this.isMoreCommitsShown
+        ? this.commits
+        : this.commits.slice(0, this.numberOfVisibleCommits);
+    },
+  },
+};
+</script>
+
+<style>
+.details-suspected-commit {
+  &__content-block {
+    padding: 13px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  &__left {
+    display: flex;
+    flex-direction: column;
+    font-size: 14px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
```

Comment:
Are all of those styles really necessary?

Reply:
no I guess, it is due to I copied this styles from zeplin

Reply:
I tried removing this but by removing it will look like 
![image](https://user-images.githubusercontent.com/23169768/121543077-43ccdf00-ca26-11eb-99a9-9cbc392c5e7b.png)